packages/sit-onyx/src/i18n/locales/en-US.json

```diff
@@ -91,9 +91,9 @@
     "toggleContextMenu": "Toggle context menu",
     "navigationHeadline": "Navigation"
   },
-  "showTooltip": {
-    "info": "Show info tooltip",
-    "error": "Show error tooltip"
+  "tooltip": {
```

Comment:
the de-DE should be adjusted too, right? there are still the old keys. ko-KR does not have the keys yet.